libbeat/tests/integration/framework.go

```diff
@@ -303,41 +443,116 @@ func EnsureESIsRunning(t *testing.T) {
 
 	esPort := os.Getenv("ES_PORT")
 	if esPort == "" {
-		esPort = "9200"
+		switch scheme {
+		case "http":
+			esPort = "9200"
+		case "https":
+			esPort = "9201"
+		}
+	}
+
+	user := os.Getenv("ES_USER")
+	if user == "" {
+		user = "admin"
+	}
+
+	pass := os.Getenv("ES_PASS")
+	if pass == "" {
+		pass = "testing"
 	}
```

Comment:
Not the scope of this PR, but I believe the best would be to have a single function that loads those credentials. That way it'd be easy to know what can or need to be set and where they come from